javascript/apps/taiga/src/app/modules/workspace/feature-detail/components/workspace-detail-projects/workspace-detail-projects.component.html

```diff
@@ -12,7 +12,7 @@
       <div class="wks-main-content-top">
```

Comment:
Workspace-detail missing:
- create your first project card placeholder
- create project button
- workspace actions button (it appears next to the workspace title)
- it still appear the 'admin' tag, is it correct?

![image](https://github.com/taigaio/taiga/assets/5970959/a974782c-3f6d-4661-ba86-df07b2dfdb03)

Reply:
Invite to workspace button is missing
![image](https://github.com/taigaio/taiga/assets/5970959/a5c7c378-6302-4407-81e6-6c80810c64a9)

Looks like the endpoint '
http://localhost:8000/api/v2/workspaces/QTya1gUXEe67nA3q1DAN0g' property userRole still being 'admin' instead of 'member':
```
{"id":"QTya1gUXEe67nA3q1DAN0g","name":"new wks","slug":"new-wks","color":2,"totalProjects":0,"hasProjects":false,"userRole":"admin"}
```